--- libvirtd.conf	2023-01-16 20:56:33.000000000 +0100
+++ /etc/libvirt/libvirtd.conf	2022-11-27 00:16:31.323397078 +0100
@@ -82,7 +82,7 @@
 # activation.
 #
 # This is restricted to 'root' by default.
-#unix_sock_group = "libvirt"
+unix_sock_group = "libvirt"
 
 # Set the UNIX socket permissions for the R/O socket. This is used
 # for monitoring VM status only
@@ -105,7 +105,7 @@
 #
 # If not using PolicyKit and setting group ownership for access
 # control, then you may want to relax this too.
-#unix_sock_rw_perms = "0770"
+unix_sock_rw_perms = "0770"
 
 # Set the UNIX socket permissions for the admin interface socket.
 #
